test/e2e/network/ingress.go

```diff
@@ -44,6 +44,7 @@ const (
 	NEGAnnotation           = "alpha.cloud.google.com/load-balancer-neg"
 	NEGUpdateTimeout        = 2 * time.Minute
 	instanceGroupAnnotation = "ingress.gcp.kubernetes.io/instance-groups"
+	statusPrefix            = "ingress.kubernetes.io"
```

Comment:
Humm..We should move all these constants to ingress_utils.go

Reply:
Done.

Ideally, we should export them from ingress-gce and reuse them here. Any idea why we dont do that?

Reply:
Sounds right, not sure why :)